models/issue_list.go

```
@@ -231,15 +237,21 @@ func (issues IssueList) loadAssignees(e Engine) error {
 			var assigneeIssue AssigneeIssue
 			err = rows.Scan(&assigneeIssue)
 			if err != nil {
-				rows.Close()
+				err = rows.Close()
+				if err != nil {
```

This if statement is not necessary because no matter the outcome err will be returned

---

I think on thoses cases we should return the first error (at end) and log the second.

---

I've defaulted to the proposal of @sapk for all these issues.